src/bindings/python/fluxacct/accounting/bank_subcommands.py

```diff
@@ -114,7 +114,7 @@ def view_bank(conn, bank, users=False):
         if rows:
             bank_str = get_bank_rows(cur, rows, bank)
         else:
-            return "Bank not found in bank_table"
+            raise ValueError("Bank not found in bank_table")
```

Comment:
Similar comment as before, it might be nice to include the failing bank name in the error message.